projects/igniteui-angular/src/lib/grids/column.component.ts

```diff
@@ -107,18 +107,31 @@ export class IgxColumnComponent implements AfterContentInit {
     @Input()
     public groupable = false;
     /**
-     * Sets/gets whether the column is editable.
+     * Gets whether the column is editable.
      * Default value is `false`.
      * ```typescript
      * let isEditable = this.column.editable;
      * ```
+     * @memberof IgxColumnComponent
+     */
+    @Input()
+    get editable(): boolean {
+        const editableInRowEdit = this.grid && this.grid.rowEditable && this.field !== this.grid.primaryKey;
```

Comment:
Why calculate the value of `editableInRowEdit` at all, if `this._editable` is defined?

Reply:
fixed.